django_sendfile/utils.py

```diff
@@ -81,23 +92,29 @@ def sendfile(request, filename, attachment=False, attachment_filename=None,
         ``False``: No ``Content-Disposition`` filename
         ``String``: Value used as filename
 
-    If neither ``mimetype`` or ``encoding`` are specified, then they will be guessed via the
-    filename (using the standard Python mimetypes module)
+    Any of ``encoding``, ``filesize and ``mimetype`` left to _guess will be guessed via the
```

Comment:
Don't make reference to `_guess` as no one reading the documentation will know what that is. Rephrase that bit to be something like "lef to their default values".